extension/js/content_scripts/webmail/generic/setup-webmail-content-script.ts

```diff
@@ -263,6 +263,7 @@ export const contentScriptSetupIfVacant = async (webmailSpecific: WebmailSpecifi
         appendAfter.after(factory.embeddedPubkey(armoredPubkey, false)); // xss-safe-value
       }
     });
+    BrowserMsg.send.setHandlerReadyForPGPBlock('broadcast');
     BrowserMsg.listen(tabId);
```

Comment:
probably `BrowserMsg.send.setHandlerReadyForPGPBlock('broadcast');` should be after `BrowserMsg.listen(tabId)`, so it'll be sent after `chrome.runtime.onMessage.addListener` is executed?

Reply:
Sounds good